config/config.go

```diff
@@ -289,6 +311,7 @@ type Config struct {
 	Deployment               DeploymentConfig         `yaml:"deployment,omitempty"`
 	Extensions               Extensions               `yaml:"extensions,omitempty"`
 	ExternalServices         ExternalServices         `yaml:"external_services,omitempty"`
+	HealthConfig             HealthConfig             `yaml:"health_config,omitempty" json:"healthConfig"`
```

Comment:
Look below in `NewConfig` : this is where I guess you should create the default config.